azure-pipelines.yml

```diff
@@ -121,6 +121,63 @@ jobs:
         ADO_PULL_REQUEST_LATEST_COMMIT: HEAD
         ADO_PULL_REQUEST_TARGET_BRANCH: $(System.PullRequest.TargetBranch)
 
+- job: getPythonVersion
+  displayName: "get Python Version"
+  pool:
+    name: 'pool-ubuntu-2004'
+  steps:
+    - task: UsePythonVersion@0
+      displayName: 'Use Python 3.10'
+      inputs:
+        versionSpec: '3.10'
+    - template: .azure-pipelines/templates/azdev_setup.yml
+    - bash: pip install wheel==0.30.0
```

Comment:
Why is wheel versionpinned?

Reply:
https://github.com/Azure/azure-cli-extensions/blob/3f8c8bf42d5bd9dfc231b747b0f76b2dfd3925e5/scripts/ci/test_index.py#L175